apps/contracts/contracts/Zengo.sol

```diff
@@ -211,12 +212,68 @@ contract ZengoDAO is PermissionsEnumerable, ContractMetadata, Constants {
         votingIteration.vote[msg.sender] = VerificationState(_vote);
         votingIteration.hasVoted[msg.sender] = true;
         votingIteration.voteCount[VerificationState(_vote)]++;
+        // TODO: trigger concludeVotingIteration when one of the
+        // consensusIteration reaches the threshold votesPercents
+        if ((votingIteration.voteCount[VerificationState(_vote)] * 100) > moderatorList.length * 51) {
```

Comment:
instead of hardcoding the `51` here, does the `THRESHOLD_VOTE_LIMIT` should be used?

Reply:
Yes...thanks for pointing it out...will update it

Reply:
Updated